lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -11,7 +11,7 @@
     name: vmware_vm_inventory
     plugin_type: inventory
     short_description: VMware Guest inventory source
-    version_added: "2.6"
+    version_added: "2.7"
```

Comment:
Not sure if this should have changed

Reply:
Actually this was typo. This plugin was added in 2.7 version and not in 2.6